pandas/core/frame.py

```diff
@@ -1144,32 +1143,58 @@ def to_gbq(self, destination_table, project_id, chunksize=10000,
         Parameters
         ----------
         dataframe : DataFrame
-            DataFrame to be written
+            DataFrame to be written to Google BigQuery.
         destination_table : string
-            Name of table to be written, in the form 'dataset.tablename'
+            Name of table to be written, in the form 'dataset.tablename'.
         project_id : str
             Google BigQuery Account project ID.
         chunksize : int (default 10000)
             Number of rows to be inserted in each chunk from the dataframe.
+            Set to ``None`` to load the whole dataframe at once.
         verbose : boolean (default True)
-            Show percentage complete
+            Show percentage complete.
         reauth : boolean (default False)
             Force Google BigQuery to reauthenticate the user. This is useful
             if multiple accounts are used.
         if_exists : {'fail', 'replace', 'append'}, default 'fail'
+            Behavior when the destination table exists.
             'fail': If table exists, do nothing.
             'replace': If table exists, drop it, recreate it, and insert data.
             'append': If table exists, insert data. Create if does not exist.
         private_key : str (optional)
             Service account private key in JSON format. Can be file path
             or string contents. This is useful for remote server
-            authentication (eg. Jupyter/IPython notebook on remote host)
-        """
+            authentication (eg. Jupyter/IPython notebook on remote host).
+        auth_local_webserver : boolean (default False)
+            Use the [local webserver flow] instead of the [console flow]
```

Comment:
I think those [..] need a trailing underscore to make them into actual links? 

(you can test with `python doc/make.py --single DataFrame.to_gbq`)

Reply:
Done in https://github.com/pandas-dev/pandas/pull/20564/commits/37b2a08af9ced867af4c969b588a9a8086a2e510

Also, thanks for the test command. Much faster than trying to build the whole docs set!